web/src/message_edit.js

```diff
@@ -1127,12 +1135,20 @@ export function save_message_row_edit($row) {
 }
 
 export function maybe_show_edit($row, id) {
+    if (message_lists.current === undefined) {
+        return;
+    }
+
     if (currently_editing_messages.has(id)) {
         message_lists.current.show_edit_message($row, currently_editing_messages.get(id));
     }
 }
 
 export function edit_last_sent_message() {
+    if (message_lists.current === undefined) {
+        return;
+    }
```

Comment:
Reading `hotkey.js`, I think there might be a subtle bug here; we don't return `false` in the one caller in the event that it wasn't processed. Likely no user-facing effect, but possibly a follow-up commit should make this function return `true/false` based on whether it did something, and the caller can read `return message_edit.edit_last_sent_message();`

Reply:
I don't know what line you are talking about here.

Reply:
So hotkey.js does `preventDefault` if the `process_key` helper returns true, and doesn't if it returns false. 

Our convention is that we `return true` if the hotkey code did something in response to the key -- and false if we decided not to process it as a hotkey -- and these are probably a case of "should not process as a hotkey".